package.json

```diff
@@ -28,12 +28,13 @@
     "build:prod": "cross-env NODE_ENV=production yarn _build",
     "build:node": "yarn _babel && yarn build:ts",
     "build:ts": "tsc -p ./config/tsconfig.json",
-    "build:test": "tsc -p ./test/tsconfig.json",
+    "build:test": "tsc -p ./test/unit/tsconfig.json",
```

Comment:
This was added as part of an early-on change when the new AVA tests compiled with a separate tsconfig. They no longer use TS at all, for now. So I could put this back. Is it worth it?